lighthouse-cli/bin.ts

```diff
@@ -229,7 +229,7 @@ function handleError(err: LightHouseError) {
   if (err.code === 'ECONNREFUSED') {
     showConnectionError();
   } else if (err.message.toLowerCase().includes('multiple tabs')) {
```

Comment:
good call, just needed the `process.exit(_RUNTIME_ERROR_CODE)` when it fails on that, didn't notice it was the same thing now.

But yeah, sorry I didn't realize that `raw.js` wouldn't work with what you were up to in #639. We could have made that PR go on even longer :) Does the simplified `checkForMultipleTabsAttached`here seem like it will still work now that we go to `about:blank` before starting? Testing it with multiple tabs in the CLI/extension seems OK.